web/src/client/storage.test.js

```diff
@@ -67,7 +87,76 @@ describe("#getProposal", () => {
       { text: "Mount /dev/sdb1 as root", subvol: false, delete: false }
     ]);
 
-    expect(proposal.volumes[0].mountPoint).toEqual("/test1");
+    expect(proposal.volumes[0]).toEqual({
+      mountPoint: "/test1",
+      optional: true,
+      deviceType: "partition",
+      encrypted: false,
+      fsTypes: ["Btrfs", "Ext3"],
+      fsType: "Btrfs",
+      minSize: 1024,
+      maxSize:2048,
+      fixedSizeLimits: false,
+      adaptiveSizes: false,
+      snapshots: true,
+      snapshotsConfigurable: true,
+      snapshotsAffectSizes: false,
+      sizeRelevantVolumes: []
+    });
     expect(proposal.volumes[1].mountPoint).toEqual("/test2");
   });
 });
+
+describe("#calculate", () => {
+  it("calculates a default proposal when no settings are given", async () => {
+    const client = new StorageClient(dbusClient);
+    await client.calculateProposal({});
+
+    expect(calculateFn).toHaveBeenCalledWith({});
```

Comment:
NP: actually, we're testing that a new proposal was requested…. but not _the default_ one.

Reply:
A defaut proposal means a proposal without settings.